tests/expectedoutputs/TestDataPostProcessor/test_DataPostProcessor_WindShear_GetFeatureInfo.json

```diff
@@ -1 +1 @@
-[{"name":"output","standard_name":"output","feature_name":"output_0","units":"m/s","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600.0":{"2019-01-01T22:00:00Z":"-0.195704"}}},{"name":"var_wind_speed_at_600","standard_name":"wind_speed","feature_name":"wind_speed_1","units":"m s-1","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600.0":{"2019-01-01T22:00:00Z":"16.896723"}}},{"name":"var_wind_speed_at_200","standard_name":"wind_speed","feature_name":"wind_speed_2","units":"m s-1","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600.0":{"2019-01-01T22:00:00Z":"16.701019"}}}]
\ No newline at end of file
+[{"name":"output","standard_name":"output","feature_name":"output_0","units":"m/s","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600":{"2019-01-01T22:00:00Z":"-0.195704"}}},{"name":"var_wind_speed_at_600","standard_name":"wind_speed","feature_name":"wind_speed_1","units":"m s-1","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600":{"2019-01-01T22:00:00Z":"16.896723"}}},{"name":"var_wind_speed_at_200","standard_name":"wind_speed","feature_name":"wind_speed_2","units":"m s-1","point":{"SRS":"EPSG:4326","coords":"3.256556,52.592725"},"dims":["elevation","time"],"data":{"600":{"2019-01-01T22:00:00Z":"16.701019"}}}]
```

Comment:
I've changed the test data so that it is now consistent for both PSQL and Sqlite. The changes are that the dimension value changes from e.g. `10.0` to `10`.